source/handy_httpd/server.d

```diff
@@ -47,6 +47,12 @@ class HttpServer {
         this.workerPool.isDaemon = true;
     }
 
+    /**
+     * Will be called before the socket is bound to the address. One can set
+     * special socket options in here by overriding it.
+     */
+    protected void configurePreBind(Socket socket) {}
```

Comment:
Looks good! However, I would like to include a small note in this ddoc snippet which explains the issue you encountered, so that other users may find the solution right away.